src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -1036,14 +1096,32 @@ exner(ts::ThermodynamicState) = exner(air_temperature(ts), air_density(ts), Phas
 
 The relative humidity, given a thermodynamic state `ts`.
 """
-function relative_humidity(ts::ThermodynamicState)
+function relative_humidity(ts::ThermodynamicState{FT}) where {FT<:Real}
   q = PhasePartition(ts)
   q_vap = q.tot - q.liq - q.ice
-  p_vap = q_vap * air_density(ts) * gas_constant_air(ts) * air_temperature(ts)
+  p_vap = q_vap * air_density(ts) * FT(R_v) * air_temperature(ts)
```

Comment:
Why don't we replace the (redundant) code here with a call to `relative_humidity(T, p, q)`, with temperature and pressure from `air_temperature`, `air_pressure`? That way we reduce possible error sources from having relative humidity defined twice in the code.

Reply:
Agreed.